### DISCUSSION TITLE
Language cannot be changed on the home page

### DISCUSSION THREAD

**Discussion by DC7IA** [Traewelling/traewelling#2055] [Ideas / Feature Requests]
**Describe the bug**
I chose bug since this affects the user experience.

To choose the language, you first have to click a link, then change the language on another page, which is confusing for user that do not understand German.

**Expected behavior**

Home page should include a way to change the language.

**Comment by HerrLevin** [comment]
Generally this shouldn't be a problem, since we're showing the preferred browser language if you're not logged in. But we could consider adding this in #1131. I'll change this ticket to enhancement.

**Comment by 0AgentSmith** [comment]
Close as outdated? 
Language selection is now available on the dashboard for both designs.
